backend/src/wamp.js

```diff
@@ -113,4 +117,16 @@ function setupWamp() {
   return wamp;
 }
 
+function wampPublish(topic, args) {
```

Comment:
It is suboptimal as this function relies on the fact that someone will call `setupWamp`, and only once. It is currently the case, and this implementation will work most of the time, but I wanted to let you know that ideally we should avoid such assumptions and dependencies on singletones.

Reply:
not very sure what do you mean by this, but I will change the whole logic and let you review